internal/task/task/bs/create_volume.go

```diff
@@ -76,6 +77,13 @@ func NewCreateVolumeTask(curveadm *cli.CurveAdm, cc *configure.ClientConfig) (*t
 	subname := fmt.Sprintf("hostname=%s image=%s", hc.GetHostname(), cc.GetContainerImage())
 	t := task.NewTask("Create Volume", subname, hc.GetSSHConfig())
 
+	if len(cc.GetContainerImage()) == 0 {
+		if err != nil {
+			return t, nil
+		}
+	} else {
```

Comment:
We should prompt based on the actual volume size and the expected, rather than the container image.